src/pages/resultsView/mutation/ResultsViewMutationTable.tsx

```diff
@@ -125,6 +136,29 @@ export default class ResultsViewMutationTable extends MutationTable<
         ].shouldExclude = () => {
             return !this.props.uniqueSampleKeyToTumorType;
         };
+
+        this._columns[MutationTableColumnType.CLONAL].shouldExclude = () => {
+            return !this.hasRequiredASCNProperty('ccfMCopies');
```

Comment:
seems like these constants are used a lot so we should definately make string enum for them.

Reply:
@sheridancbio rename?

Reply:
I've look up enums in the typescript documentation .. and string constant enums are explicitly supported.  I may personally quibble with calling a set of string constants an "enum", but I guess that is the new lingo we are working with.